demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -2202,7 +2306,7 @@ export class DemoMeetingApp
 
     const transcriptSpan = document.createElement('span') as HTMLSpanElement;
     transcriptSpan.classList.add('transcript-content');
-    transcriptSpan.innerText = segment.content;
+    transcriptSpan.innerHTML = segment.content;
```

Comment:
Same here, any reason we need to set `innerHTML` instead?

Reply:
The content contains html rich text(some words in content based on whether they are PII identified or low confidence words). Using innerText will output string in plain text something like this: "My name is <span class='entity-color'>Akash</span>" instead of actually displaying the word Akash in color.

Reply:
@akashuc are you saying that a segment delivered over this push channel from Transcribe will include raw HTML, which we then inline directly into the page?

Reply:
@richnew10 Not sure if I understood your question here. But Transcribe is sending words in string format. We need to show the PII identified and low confidence words in demo app using color coding and underline respectively. The segment here is formed by concatenating those words that transcribe sends over. In this segment, we need to highlight/underline just those words that fits the criteria mentioned. So once we are forming the segment in demo app, we would be having a mix of plain text as well as html rich text.

Reply:
Oh, I see where in this file you're constructing the spans. You should probably do something a little more sophisticated here.

Reply:
Can you check Amazon Transcribe console implementation. It seems that they are just identifying PII content in finalized results which can simplify the UI implementation. The package name is `WolverineConsole-Assets`.